src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -63,12 +63,12 @@ $pf-v6--include-toolbar-breakpoints: true !default;
   --#{$toolbar}__item--m-label--FontWeight: var(--pf-t--global--font--weight--body--bold);
 }
 
-// - Toolbar content - Toolbar group - Toolbar item - Toolbar group chip container - Toolbar group chip group
+// - Toolbar content - Toolbar group - Toolbar item - Toolbar group label container - Toolbar group label group
 .#{$toolbar}__content-section,
 .#{$toolbar}__group,
 .#{$toolbar}__item,
-.#{$toolbar}__group.pf-m-chip-group-container,
-.#{$toolbar}__group.pf-m-chip-group {
+.#{$toolbar}__group.pf-m-label-group-container,
```

Comment:
Wouldn't this conflict with line 235, since they're both targeting a `.toolbar-group.label-group-container` selector?